tests/test-update-refspec-checkpoint.c

```diff
@@ -1491,6 +1491,8 @@ test_update_refspec_checkpoint_eos3a_eos4 (EosUpdaterFixture *fixture,
       { NULL, NULL, "ASUSTeK COMPUTER INC.", "Z550MA", FALSE, NULL, NULL, NULL, FALSE, FALSE },
       { NULL, NULL, "Endless", "ELT-JWM", FALSE, NULL, NULL, NULL, FALSE, FALSE },
       { NULL, NULL, "Endless", "ELT-JWM", FALSE, NULL, NULL, NULL, TRUE, TRUE },
+      { "latest1", NULL, "Endless", "EE-200", FALSE, NULL, NULL, NULL, FALSE, FALSE },
+      { "latest1", NULL, "Endless", "EE-200", FALSE, NULL, NULL, NULL, TRUE, TRUE },
```

Comment:
Unfortunately, my request to make you change the suffix match to `/latest1` broke these tests. A ref starting with `/` is illegal, so I think you'll have to use something more specific like `os/eos/amd64/latest1`.

Reply:
Oops!  My fault.